bin/oneoff/nces_data/import_school_stats_by_years_2020_2021_ccd

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../dashboard/config/environment'
+
+CDO.log = Logger.new(STDOUT)
+
+SURVEY_YEAR = '2020-2021'.freeze
+
+DRY_RUN = true
+
+VIRTUAL_SCHOOL_MAP = {
+  'Full Virtual' => 'Yes',
+  'Not Virtual' => 'No',
+  'Supplemental Virtual' => 'No',
+  'Virtual with face to face options' => 'Yes',
+  '–' => nil,
+  '†' => nil
```

Comment:
Why do we need these characters in the hashes?

Reply:
They're in the raw data as their own values, do you have an example file, clare?

Reply:
Yes! Here are the files in [drive](https://drive.google.com/drive/u/0/folders/1mNba3MCy3RpyDJDRBI7q1YPDC-cD4v5q) which explain the symbols. (I removed the extra rows at the top and bottom before uploading them to S3)